### PR TITLE
Update stac-load-e84-aws.py

### PR DESCRIPTION
**Pull request by mehran66**
the old link and parameters do not work anymore

### CODE REVIEW

**Comment by Kirill888** [comment]
Thanks for this @mehran66, looks like with the new collection we can drop that `cfg`  overrides as stac metadata now contains all the needed information. Also, please run `black` on the file, pipeline complains about it.

File: notebooks/stac-load-e84-aws.py [hunk 1]
```diff
@@ -78,10 +78,10 @@ def convert_bounds(bbox, invert_y=False):
 r = 100 * km2deg
 bbox = (x - r, y - r, x + r, y + r)
 
-catalog = Client.open("https://earth-search.aws.element84.com/v0")
+catalog = Client.open("https://earth-search.aws.element84.com/v1/")
 
 query = catalog.search(
-    collections=["sentinel-s2-l2a-cogs"], datetime="2021-09-16", limit=100, bbox=bbox
+    collections=["sentinel-2-l2a"], datetime="2020-01-01/2020-02-31", limit=100, bbox=bbox
```
Comment (Kirill888) [inline notebooks/stac-load-e84-aws.py]: Why was change in date necessary? This one queries 2 months worth of data, a bit too much for github pipelines.

Comment (mehran66) [reply notebooks/stac-load-e84-aws.py]: Thanks @Kirill888 Fixed date, removed config, and made a few more changes. Ran the code locally and it runs all fine.

**Review by Kirill888 — APPROVED** [review]
thanks

FYI: docs website won't update straight away, publishing of notebooks appears to be broken right now, I'll need to investigate.

**Pull request merged**